src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -229,29 +243,82 @@ public boolean isThrottleMatrixConfigurations() {
         assert category != null && !category.equals("");
         List<Queue.Task> categoryTasks = new ArrayList<Queue.Task>();
         Collection<ThrottleJobProperty> properties;
-        DescriptorImpl descriptor = Jenkins.getActiveInstance().getDescriptorByType(DescriptorImpl.class);    
+        DescriptorImpl descriptor = Jenkins.getActiveInstance().getDescriptorByType(DescriptorImpl.class);
```

Comment:
Please avoid gratuitous whitespace changes. Configure your editor to not make them.